src/TMS/Approval/Approval.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
```

Comment:
Bitte die declare Anweisung benutzen https://concepts-and-training.pixsoftware.de/confluence/display/ILIAS/PSR-12+Code+Formatierung